lib/streams/vinyl-convert-stream.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const stream = require('stream');
+
+const Vinyl = require('vinyl');
+
+/**
+ * Vinyl converter Transform stream.
+ *
+ * Stream convert chunk to vinyl-object.
+ *
+ * Chunk should has a "path" property.
+ *
+ * @extends VinylConvertStream
```

Comment:
IMHO, you should use `@extends stream.Transform` instead of `@extends VinylConvertStream`.